.travis.yml

```diff
@@ -17,8 +18,13 @@ env:
     - TOX_ENV=django111
```

Comment:
The tests should also run on django20 & django21 (and fail but that's ok). But if this was added here... it would try to run py27-django20 which is unecessary. For the tests to run in parallel and cleanly control which interpreter is used, I find it more convenient and maintainable to create an explicit matrix instead of relying on the matrix built from python, env etc. For instance:

```
matrix:
  include:
    - python: 2.7
      env: TOXENV=django111
    - python: 3.6
      env: TOXENV=django111
    - python: 3.7
      env: TOXENV=django111
    - python: 3.6
      env: TOXENV=django20
    - python: 3.6
      env: TOXENV=django21
    - python: 3.7
      env: TOXENV=django20
    - python: 3.7
      env: TOXENV=django21
    - python: 3.5
      env: TOXENV=quality
    - python: 3.6
      env: TOXENV=docs
```